File: ant-sdk/antd-cpp/examples/01-connect.cpp
#include <iostream>
#include "antd/antd.hpp"

int main() {
    try {
        antd::Client client;  // defaults to http://localhost:8082

        auto health = client.health();
        std::cout << "OK: " << (health.ok ? "true" : "false") << "\n";
        std::cout << "Network: " << health.network << "\n";
    } catch (const antd::AntdError& e) {
        std::cerr << "Error: " << e.what() << "\n";
        return 1;
    }
    return 0;
}

File: ant-sdk/antd-php/examples/01-connect.php
<?php

/**
 * Example 01: Connect to antd and check health.
 *
 * Prerequisites: antd daemon running (ant dev start)
 */

require_once __DIR__ . '/../vendor/autoload.php';

use Autonomi\Antd\AntdClient;

$client = new AntdClient('http://localhost:8082');

$health = $client->health();
echo "OK: " . ($health->ok ? 'true' : 'false') . "\n";
echo "Network: {$health->network}\n";
